pom.xml

```diff
@@ -1224,6 +1224,13 @@
                 <version>3.18.1</version>
             </dependency>
 
+            <dependency>
+                <!-- pinned for transitive dependencies; TODO: move this to airbase -->
+                <groupId>com.fasterxml.jackson.dataformat</groupId>
+                <artifactId>jackson-dataformat-xml</artifactId>
```

Comment:
do we need this for every new transitive dependency we pull? Does jackson provide a bom that we can import?

Reply:
Actually, they do! But I would consider this to be temporary and doing the proper solution in Airbase - I'll update airlift/airbase#346 to use it. Thanks for pointing this out!